bindings/csharp/Keystone.Net/NativeInterop.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace Keystone
+{
+    /// <summary>
+    ///   Imported symbols for interop with keystone.dll.
+    /// </summary>
+    internal class NativeInterop
+    {
+        // This shouldn't be needed, even on Windows
```

Comment:
My computer did not need it on Windows / AnyCPU. Furthermore, I'm pretty sure it will make it crash on non-Windows platforms. Finally, [LLVMSharp](https://github.com/Microsoft/LLVMSharp) does not need any such initializer, so I doubt it's really useful now.

Reply:
Since it's "AnyCPU", since you're running it on an x86_64 platform, chances are your app is ngen'd into 64 bit machine code so if, for instance, you have a 64 bit version of your keystone.dll in your output dir, everything works correctly.

Say you've built your app with AnyCPU and your app is executed on a 32 bit platform, the application is ngen'd into 32 bit machine code, so this will no longer work because you have the 64 bit version of your app.

Reply:
I unfortunately cannot test it, but that seems really strange to me, especially since I don't know any other binding library that requires this. If you want, we could add an opt-in static method that does this, in case it is needed?

Reply:
If other libraries doesn't use something like that, they leave to whoever build the final package the burden to choose the right "native" dll  (x86/x64), and that's a problem with AnyCPU code because you don't know if it will be x86 or x64 until runtime. But that's fine I guess. 
If the choice is between portability and supporting this edge case, portability wins.